addons/mail/static/src/components/thread_viewer/thread_viewer.xml

```diff
@@ -32,6 +32,7 @@
                         hasCurrentPartnerAvatar="props.hasComposerCurrentPartnerAvatar"
                         hasSendButton="props.hasComposerSendButton"
                         hasThreadTyping="props.hasComposerThreadTyping"
+                        isCompact="threadViewer.thread.model === 'mail.channel' and threadViewer.thread.mass_mailing ? false : undefined"
```

Comment:
I prefer parenthesis when priority of operations is not obvious (`and` vs `? :` first?) but otherwise ok